.env.development.local

```diff
@@ -1 +0,0 @@
-FAST_REFRESH=true
```

Comment:
Why are we removing this? :)

Reply:
I thought each of us can keep this local and not add to .github

Reply:
Makes sense tbh; I'm ok with pushing `.env.development.local` in git. I usually add `.env.example` to my repos for faster scaffolding.